### PR TITLE
how-to/deploy: update TiDB version info in deployment

### PR DESCRIPTION
**Pull request by liubo0127**
This PR updates the TiDB version info in the Ansible deployment document of `dev`, `v3.0`, and `v2.1`.

@shenli @lilin90  PTAL

### CODE REVIEW

File: v2.1/how-to/deploy/orchestrated/ansible.md [hunk 1]
```diff
@@ -123,32 +123,20 @@ The key's randomart image is:
...
-    $ git clone https://github.com/pingcap/tidb-ansible.git
-    ```
+```
+$ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
+```
 
 > **注意：**
 >
-> 请务必按文档操作，将 `tidb-ansible` 下载到 `/home/tidb` 目录下，权限为 `tidb` 用户，不要下载到 `/root` 下，否则会遇到权限问题。
+> - `$tag` 替换为选定的 TAG 版本的值
```
Comment (lilin90) [inline v2.1/how-to/deploy/orchestrated/ansible.md]:
```suggestion
> - `$tag` 替换为选定的 TAG 版本的值，例如 `v2.1.15`。
```

File: v3.0/how-to/deploy/orchestrated/ansible.md [hunk 2]
```diff
@@ -120,36 +120,17 @@ The key's randomart image is:
...
-    $ git clone https://github.com/pingcap/tidb-ansible.git
-    ```
+```
+$ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
+```
 
 > **注意：**
 >
-> 请务必按文档操作，将 `tidb-ansible` 下载到 `/home/tidb` 目录下，权限为 `tidb` 用户，不要下载到 `/root` 下，否则会遇到权限问题。
+> - `$tag` 替换为选定的 TAG 版本的值
```
Comment (lilin90) [inline v3.0/how-to/deploy/orchestrated/ansible.md]:
```suggestion
> - `$tag` 替换为选定的 TAG 版本的值，例如 `v3.0.2`。
```

File: dev/how-to/deploy/orchestrated/ansible.md [hunk 3]
```diff
@@ -119,36 +119,16 @@ The key's randomart image is:
...
-    ```
-
-- 下载 latest 版本对应的 tidb-ansible：
-
-    ```
-    $ git clone https://github.com/pingcap/tidb-ansible.git
-    ```
+```
+$ git clone https://github.com/pingcap/tidb-ansible.git
+```
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/ansible.md]: @dcalvin What do you think of removing description about previous versions?

File: v3.0/how-to/deploy/orchestrated/ansible.md [hunk 4]
```diff
@@ -429,13 +410,13 @@ location_labels = ["host"]
 
 - 服务配置文件参数调整
 
-    1.  多实例情况下，需要修改 `tidb-ansible/conf/tikv.yml` 中的 `block-cache-size` 参数:
-        - `rocksdb defaultcf block-cache-size(GB)` = MEM * 80% / TiKV 实例数量 * 30%
-        - `rocksdb writecf block-cache-size(GB)` = MEM * 80% / TiKV 实例数量 * 45%
-        - `rocksdb lockcf block-cache-size(GB)` = MEM * 80% / TiKV 实例数量 * 2.5% (最小 128 MB)
-        - `raftdb defaultcf block-cache-size(GB)` = MEM * 80% / TiKV 实例数量 * 2.5% (最小 128 MB)
+    1. 多实例情况下，需要修改 `tidb-ansible/conf/tikv.yml` 中的 `block-cache-size` 参数:
+        - `roksdb defaultcf block-cache-size(GB)` = MEM \* 80% / TiKV 实例数量 * 30%
```
Comment (dcalvin) [inline v3.0/how-to/deploy/orchestrated/ansible.md]: `roksdb`? is this right?

File: dev/how-to/deploy/orchestrated/ansible.md (hunk 3 above)
Comment (dcalvin) [reply dev/how-to/deploy/orchestrated/ansible.md]: I think it's the right thing to do. Not only making the doc shorter, but also adaptive the specific version

File: v3.0/how-to/deploy/orchestrated/ansible.md (hunk 4 above)
Comment (lilin90) [reply v3.0/how-to/deploy/orchestrated/ansible.md]: @liubo0127 I think it must be `rocksdb`.

**Pull request merged**